docs/release-notes/artifacts/pr0255.yaml

```
@@ -0,0 +1,14 @@
+schema_version: 1
+changes:
+  - title: Handle optional TLS information in charm and state
+    author: tphan025
+    type: bugfix
+    description: |
+      Allow TLSInformation.from_charm to return None when certificates are not required,
+      and handle this case in the charm code to avoid errors.
```

We should focus on user experience rather than implementation details in the release notes. Maybe something like:

```suggestion
      Fixing a bug on how certificates where handled when not required (for TCP routes).
```